docs/self-managed/setup/deploy/amazon/amazon-eks/dual-region.md

```diff
@@ -9,10 +9,10 @@ description: "Deploy two Amazon Kubernetes (EKS) clusters with Terraform for a p
 import CoreDNSKubeDNS from "./assets/core-dns-kube-dns.svg"
 
 :::warning
-Review our [dual-region concept documentation](./../../../../concepts/multi-region/dual-region.md) before continuing to understand the current limitations and restrictions of this setup, as well as the disclaimer concerning support from Camunda.
+Review our [dual-region concept documentation](./../../../../concepts/multi-region/dual-region.md) before continuing to understand the current limitations and restrictions of this blueprint setup, as well as the disclaimer concerning support from Camunda.
```

Comment:
>  as well as the disclaimer concerning support from Camunda.
Which disclaimer would this refer to now?

Reply:
removed it, was a leftover, thanks for noticing 👍